core/src/bitcoin/BitcoinLikeAddress.hpp

```diff
@@ -87,6 +86,11 @@ namespace ledger {
                                                                const api::Currency &currency,
                                                                const std::string &keychainEngine);
 
+        protected:
```

Comment:
Why protected ? The derived class can use directly `toBech32()` and `toBase58()`. I guess these 2 impl methods can be private.

Other note (more personal habit), for private impl methods, I use to define them only in an unnamed namespace in .cpp file. This avoid adding lines in the header that are in fact almost useless for lib users, and in some cases can fasten compilation.